zim/gui/widgets.py

```diff
@@ -1998,7 +1998,7 @@ def update_completion(self):
 
 		text = self.get_text()
 		if self._current_completion:
-			if text.startswith(self._current_completion) \
```

Comment:
Don't think that is right - only line where it is to a value other than `None` is here: https://github.com/Osndok/zim-desktop-wiki/blob/5dcb1500acdd5099423e46ca5161efc840320215/zim/gui/widgets.py#L2041

So should always be either the latest input text, or `None`. Have to admit that the variable name is confusing here.